bin/cache.cpp

```diff
@@ -0,0 +1,104 @@
+#include <mbgl/storage/default_file_source.hpp>
+#include <mbgl/storage/resource.hpp>
+#include <mbgl/util/chrono.hpp>
+#include <mbgl/util/run_loop.hpp>
+
+#include <args.hxx>
+
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+std::shared_ptr<std::string> readFile(const std::string& fileName) {
```

Comment:
You could use mapbox-io here instead: https://github.com/mapbox/mapbox-base/tree/master/mapbox/io

Reply:
@brunoabinader thanks for the heads-up.